5.5/test/run

```diff
@@ -89,6 +89,39 @@ function create_container() {
   echo "Created container $(cat $cidfile)"
 }
 
+function run_replication_test() {
```

Comment:
can we get a test case that 
1) launches the image using a local volume (docker run, not in openshift)
2) shuts it down
3) launches it using a new password
4) confirms the new password now works and the old one no longer does?